### PR TITLE
virtio terminal: Use Test::Warnings

### PR DESCRIPTION
**Pull request by richiejp**
Remove custom code to catch warnings and use Test::Warnings instead.

https://github.com/os-autoinst/os-autoinst/pull/721

### CODE REVIEW

**Comment by okurz** [comment]
checks fail because cpanfile does not specify `Test::Warnings` so should be rebased on https://github.com/os-autoinst/os-autoinst/pull/721 after that is merged

**Comment by richiejp** [comment]
Or you can accept my PR on your repo: https://github.com/okurz/os-autoinst/pull/1 and close this one.

Never tried it before so not sure how well it works.

**Comment by coolo** [comment]
So revoke?

**Comment by okurz** [comment]
@richiejp any chance to fix it?

File: cpanfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ requires 'Term::ANSIColor';
 requires 'Test::Compile';
 requires 'Test::More';
 requires 'Test::Simple';
+requires 'Test::Warnings';
```
Comment (aaannz) [inline cpanfile]: The master already has this added into `on 'test'` section. Seems like you rebase is not up to date.

**Pull request merged**